webpack.config.amd.dev.js

```diff
@@ -137,62 +139,65 @@ module.exports = (env, argv) => ({
             loader: 'css-loader',
             options: {
               importLoaders: 2,
-              localIdentName: '[name]_[local]-[hash:base64:5]',
-              minimize: false,
```

Comment:
starting from css-loader@1.0.0 minimize can not be used. In release note they recommend using cssnano with postcss-loader BUT later there was an issue where it was not recommended to use cssnano https://github.com/cssnano/cssnano/issues/659.
Since ```minimize``` was ```false``` I decided that there is no need in minimizing plugin